packages/extension/src/views/Events/EventItem/formatInteger.ts

```diff
@@ -0,0 +1,12 @@
+export function formatInteger(integer: number | string): string {
+  const value = integer.toString()
```

Comment:
Why not simply:
`new Intl.NumberFormat('en-US').format(1000000)`

Currently:
`formatInteger('-10.03') === "-10,.03"`

Reply:
You can't use Intl.NumberFormat because the numbers can be outside of the safe range of 2^53

Reply:
Also `formatInteger` formats integers not floats :)